### PR TITLE
Update the manifest to call http URL instead of typha health check command

### PR DESCRIPTION
**Pull request by roshanirathi**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 3d7fb00a012359e8c8f9bf05cee961e17aa1a046

https://deploy-preview-2482--calico.netlify.com

File: _includes/master/manifests/calico-typha.yaml [hunk 1]
```diff
@@ -92,17 +92,13 @@ spec:
         livenessProbe:
           exec:
             command:
-            - calico-typha
-            - check
-            - liveness
+            - curl https://127.0.0.1:5473/liveness
```
Comment (fasaxc) [inline _includes/master/manifests/calico-typha.yaml]: This shouldn't use curl here, it should use k8s' native HTTP readiness probe.  

See the example here: https://github.com/projectcalico/calico/blob/master/_includes/master/manifests/calico-node.yaml#L307

File: _includes/master/manifests/calico-typha.yaml [hunk 2]
```diff
@@ -90,19 +90,17 @@ spec:
...
           #  value: "9093"
         livenessProbe:
-          exec:
-            command:
-            - calico-typha
-            - check
-            - liveness
+          httpGet:
+            path: /liveness
+            port: 5473
```
Comment (fasaxc) [inline _includes/master/manifests/calico-typha.yaml]: `5473` is Typha's main port but the healthcheck is on port `9098`

Comment (roshanirathi) [reply _includes/master/manifests/calico-typha.yaml]: Ack

**Pull request merged**